resources/resincli.robot

```diff
@@ -69,6 +69,17 @@ Device ${device_uuid} log should contain ${value}
     Process ${result}
     Should Contain    ${result.stdout}    ${value}
 
+Adding ENV variable ${variable_name} with value ${variable_value} to application ${application_name} 
+    ${result} =  Run Process    resin   env     add     ${variable_name}    ${variable_value}    -a  ${application_name}
+    Process ${result}
+
+Checking if ENV variable is present in application ${application_name} 
```

Comment:
How about the following instead?
Check if ENV variable ${variable_name} with value ${variable_value} is present in application ${application_name}